recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
-{% set version = "20180414" %}
+{% set version = "20210325" %}
 
 package:
   name: texlive-core
   version: {{ version }}
 
 source:
-  url: ftp://tug.org/texlive/historic/2018/texlive-{{ version }}-source.tar.xz
-  sha256: fe0036d5f66708ad973cdc4e413c0bb9ee2385224481f7b0fb229700a0891e4e
-  patches:
-    - 0001-support-poppler-0.64.0.patch
+  url: https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2021/texlive-{{ version }}-source.tar.xz
+  sha256: 7aefd96608d72061970f2d73f275be5648ea8ae815af073016d3106acc0d584b
+#  patches:
+#    - 0001-support-poppler-0.64.0.patch
```

Comment:
You can probably remove these lines and the patch.

Reply:
@ocefpaf left them just in case, but yes, they can go. I will edit soon. The runs complete successfully (albeit the aarch64 and ppc runs take ~3 hours)

Reply:
Also consider adding yourself as a maintainer here if you feel like it. We could use the help :grimacing:

Reply:
Cool! Do we need the other patch? It doesn't look like it would be used in this. I can clean this up further when I try to add MacOS ARM64 support. It seems there might be a bigger issue than simple turning on/off a switch. For now, let's go with this simple upgrade and then we can tackle the OSX ARM64 stuff separately. 

@conda-forge-admin, please rerender

Reply:
To be honest I don't know about the patch. I haven't been able to maintain this recipe for a while, sorry.

Reply:
The patch had come from PR ( https://github.com/conda-forge/texlive-core-feedstock/pull/30 ). Maybe it had something to do with adding `ppc64le`?

cc @jayfurmanek